Trev8939/raster layer geo package

TADraeseke opened the pull request:
Add a sample demonstrating use of the raster layer geo package.

shellygill commented on java/raster-layer-geopackage/README.md:
```diff
@@ -0,0 +1,29 @@
+# Raster Layer GeoPackage
+
+This sample demonstrates how to open a GeoPackage, obtain a raster from the package, and display the table as a RasterLayer.
```
Format class names here?

shellygill commented on java/raster-layer-geopackage/src/main/java/com/esri/arcgisruntime/sample/rasterlayergeopackage/MainActivity.java:
```diff
@@ -0,0 +1,113 @@
+    // inflate MapView from layout
+    mMapView = findViewById(R.id.mapView);
+
+    // create a map with the BasemapType topographic
```
Comment about basemap type does not match code.

shellygill commented on java/raster-layer-geopackage/src/main/java/com/esri/arcgisruntime/sample/rasterlayergeopackage/MainActivity.java:
```diff
@@ -0,0 +1,113 @@
+        // read raster images and get the first one
+        Raster geoPackageRaster = geoPackage.getGeoPackageRasters().get(0);
+
+        // make sure an image was found in the package
```
If there's no image in the package, I'd expect an exception will already have been thrown by this point - as the raster at index 0 has already been accessed in the line above.

TADraeseke replied:
Hmm, good point...

TADraeseke commented:
@shellygill - Thanks! Updated as per your comments.

As per your third comment, I've added a check 
`if (!geoPackage.getGeoPackageRasters().isEmpty()) {`
before the `.get()`. Think this resolves the silly bit of code I wrote earlier.
